deviceadvisor/script/DATestRun.py

```diff
@@ -0,0 +1,224 @@
+import boto3
```

Comment:
Can we look into possibly some shared code refactoring so that the common device/cert/etc... management is a single script somewhere (builder? or dunno) rather than repeating it in each repo?  Language-specific invocation details would still need to be per-repo.

Reply:
That actually would be better. We can add a da test step into builder if possible. I will need to look into the crt-builder for further discussion.